iina/AppData.swift

```diff
@@ -79,6 +79,7 @@ struct Constants {
     static let subDelay = "Subtitle Delay"
     static let pip = NSLocalizedString("menu.pip", comment: "Enter Picture-in-Picture")
     static let exitPIP = NSLocalizedString("menu.exit_pip", comment: "Exit Picture-in-Picture")
+    static let custom = NSLocalizedString("Custom...", comment: "Custom")
```

Comment:
Please rename this key `menu.crop_custom` and add corresponding localizations to Localizable.strings (Base and English).